### ISSUE TITLE
[Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28657]
#### VFS team name
Claims & Appeals

#### VFS product name
Higher Level Review

#### Point of Contact/Reviewers
Josh Kim

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
"legacy appeals" link does not provide enough description for users to know its purpose and destination. 

### Supporting Guidance
[08.03.1](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
[WCAG SC 2.4.4_A](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context)

---

### Platform Recommendation
Consider using hint text below the label to provide a link for users to learn more about legacy appeals. For example:
- **[Label]** Is this claim going through legacy appeals?
- **[Hint text below, full link]** Learn more about legacy appeals

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by Mottie** [comment]
Fix has been merged and should be available in staging soon. Oddly enough, it's already available in production (see https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/frontend-support-dashboard/)

**Comment by shiragoodman** [comment]
adding labels so ticket is consistent with post-pilot process

**Comment by Mottie** [comment]
Closing this as HLR v2 wizard no longer shows the legacy question
